DS/DSAssignment5/LinkListInserction.java

```diff
@@ -0,0 +1,25 @@
+import java.util.LinkedList;
+import java.util.List;
+
+public class LinkListInserction {
+
+	public List<Integer> list = new LinkedList<Integer>();
+
+	public void insert(int item, int index) {
```

Comment:
ACTION: What is the use of 'index'? The wants to insert some number in a sorted list means after insertion the list is still expected to be sorted therefore he does not bother or rather does not know where the new number should be inserted.